appformer-form-modeler/appformer-form-modeler-codegen/appformer-form-modeler-codegen-impl/src/main/java/org/kie/appformer/formmodeler/codegen/view/impl/java/RoasterViewSourceGenerator.java

```diff
@@ -85,7 +86,7 @@ protected void addBaseViewFieldsAndMethodImpls( final SourceGenerationContext co
 
         for ( final FieldDefinition fieldDefinition : context.getFormDefinition().getFields() ) {
 
-            if ((fieldDefinition.isAnnotatedId() && !displaysId()) || isBanned( fieldDefinition ))
+            if (isBanned( fieldDefinition ))
```

Comment:
@pefernan It is a small thing, but I have noticed this particular line is not tested in the unit tests...